packages/salesforcedx-vscode-core/src/commands/util/sourcePathStrategies.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
```

Comment:
The new home for SourcePathStrategy related code since it is now shared between the template commands and retrieve component command.